lightly_studio/src/lightly_studio/core/dataset.py

```diff
@@ -153,7 +153,7 @@ def load_or_create(
         if dataset.sample_type != sample_type:
             raise ValueError(
                 f"Dataset with name '{name}' already exists with sample type "
-                f"'{dataset.sample_type}', but '{sample_type}' was requested."
+                f"'{dataset.sample_type.value}', but '{sample_type.value}' was requested."
```

Comment:
Otherwise on python 3.13
```bash
E AssertionError: Regex pattern did not match.
E Regex: "is having sample type 'video', expected 'image'"
E Input: "[...] is having sample type 'SampleType.VIDEO', expected 'SampleType.IMAGE'."
```